### PR TITLE
[HUE] Add configuration examples for Alert & Effect

### LINKED ISSUE #3573: [Hue] non working blink (none, alert, la), WARN in the log
At this moment the blink does not seem to work, however I believe I remember to have used in the past, any hints?

The log contains the following

```
2017-06-03 09:19:32.675 [WARN ] [thome.io.rest.core.item.ItemResource] - Received HTTP POST request at 'items/HueLiving_alert' with an invalid status value 'SELECT'.
2017-06-03 09:19:35.315 [WARN ] [thome.io.rest.core.item.ItemResource] - Received HTTP POST request at 'items/HueLiving_alert' with an invalid status value 'LSELECT'.
2017-06-03 09:19:38.467 [WARN ] [thome.io.rest.core.item.ItemResource] - Received HTTP POST request at 'items/HueLiving_alert' with an invalid status value 'NONE'.
```

### PR DESCRIPTION
**Pull request by martinvw**
Fixes #3573

Signed-off-by: Martin van Wingerden <martinvw@mtin.nl>

### CODE REVIEW

File: extensions/binding/org.eclipse.smarthome.binding.hue/README.md [hunk 1]
```diff
@@ -125,6 +127,8 @@ sitemap demo label="Main Menu"
 		Slider item=		Light1_Dimmer
 		Colorpicker item=	Light1_Color
 		Slider item=		Light1_ColorTemp
+		Switch item=	    Light1_Alert mappings=[NONE="None", SELECT="Alert", LSELECT="Long Alert"]
```
Comment (kaikreuzer) [inline extensions/binding/org.eclipse.smarthome.binding.hue/README.md]: the "old" text used tabs here - I do not really care whether its spaces or tabs in the readme, but it should be consistent.

Comment (martinvw) [inline extensions/binding/org.eclipse.smarthome.binding.hue/README.md]: Agree I think my idea was acting 'smart' there

File: extensions/binding/org.eclipse.smarthome.binding.hue/README.md [hunk 2]
```diff
@@ -105,6 +105,8 @@ Switch	Light1_Toggle		{ channel="hue:0210:1:bulb1:color" }
 Dimmer  Light1_Dimmer		{ channel="hue:0210:1:bulb1:color" }
 Color 	Light1_Color		{ channel="hue:0210:1:bulb1:color" }
 Dimmer 	Light1_ColorTemp	{ channel="hue:0210:1:bulb1:color_temperature" }
```
Comment (martinvw) [inline extensions/binding/org.eclipse.smarthome.binding.hue/README.md]: I think the same happened here, I ll check it once I m back at home.

File: extensions/binding/org.eclipse.smarthome.binding.hue/README.md [hunk 3]
```diff
@@ -102,14 +102,16 @@ Bridge hue:bridge:1 [ ipAddress="192.168.0.64" ] {
...
+Dimmer	Light1_ColorTemp	{ channel="hue:0210:1:bulb1:color_temperature" }
+String	Light1_Alert		{ channel="hue:0210:1:bulb1:alert" }
+Switch	Light1_Effect		{ channel="hue:0210:1:bulb1:effect" }
 
 // Bulb2
-Switch	Light2_Toggle		{channel="hue:0220:1:bulb2:brightness"}				
-Dimmer	Light2_Dimm		{channel="hue:0220:1:bulb2:brightness"}
-Dimmer	Light2_ColorTemp	{channel="hue:0220:1:bulb2:color_temperature"}
+Switch	Light2_Toggle		{ channel="hue:0220:1:bulb2:brightness" }
+Dimmer	Light2_Dimm		{ channel="hue:0220:1:bulb2:brightness" }
```
Comment (martinvw) [inline extensions/binding/org.eclipse.smarthome.binding.hue/README.md]: The only bad thing about this, is that it renders incorrect if you have configure your tab-width to be 4 chars.

**Comment by martinvw** [comment]
Made everything consistent, however had to make it look bad at my own pc because GitHub uses a tab-width of 8.

**Pull request merged**